src/gui/properties/proptabbar.cpp

```diff
@@ -63,7 +63,7 @@ PropTabBar::PropTabBar(QWidget *parent)
     // Peers tab
     QPushButton *peersButton = new QPushButton(
 #ifndef Q_OS_MACOS
-            UIThemeManager::instance()->getIcon(u"edit-find-user"_qs),
+            UIThemeManager::instance()->getIcon(u"peers-user"_qs),
```

Comment:
Just "peers"?
Also maybe change "office-chart-line" to "chart-line(s)"?

Reply:
Check.